src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return (days * 40) - 50;
+  } else if (days >= 3) {
+    return (days * 40) - 20;
+  } else {
+    return days * 40;
```

Comment:
```suggestion
  }
    return days * 40;
```
No need in this else. This case will be returned automatically if both previous ones will fail

Reply:
fixed it !)